palindrome.c

```diff
@@ -0,0 +1,38 @@
+//Ashank Kumar, program checks if a string is a palindrome
+#include <stdio.h>
```

Comment:
Nice job overall, but you didn't handle the empty string or capitals. The empty string actually returns a segmentation fault; it should return that it is a palindrome. Hannah returns that it's not a palindrome when it is.